File: WebApplication/src/modules/report/models/sheet.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { SheetRowDto } from "./sheet-row.dto";
import { SheetYearDto } from "./sheet-year.dto";

export class SheetDto extends SheetRowDto {
  id: number;
  year: NomenclatureDto;
  bank: NomenclatureDto;

  sheetList: SheetYearDto[] = [];
}

File: StudentCredit.Application/Reports/Dtos/SummaryReports/SheetDto.cs
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Reports.Dtos.SummaryReports
{
    public class SheetDto : SheetRowDto
    {
        public int Id { get; set; }

        public NomenclatureDto Year { get; set; }

        public NomenclatureDto Bank { get; set; }

        public decimal? Limit { get; set; }

        public decimal? FulfillmentOfTheLimit { get; set; }

        public List<SheetYearDto> SheetList { get; set; } = new List<SheetYearDto>();
    }
}
